src/tespy/connections.py

```diff
@@ -596,6 +597,27 @@ def to_flow_design(self):
         """
         return [self.m.design, self.p.design, self.h.design, self.fluid.design]
 
+    def get_physical_exergy(self, pamb, Tamb):
+        r"""
+        Get the value of a connection's specific physical exergy.
+        Calcute physical exergy of connection
+
+        Parameters
+        ----------
+        pamb : float
+            Ambient pressure pamb / Pa.
+
+        Tamb : float
+            Ambient temperature Tamb / K.
+        
+        Note
+        ----
+            .. math::
+
+                E^{\text{PH}} = \dot{m} \cdot e^{\text{PH}}
+        """
+        self.ex_physical = calc_physical_exergy(self, pamb, Tamb)
+        self.Ex_Physical = self.m.val_SI * self.ex_physical
```

Comment:
This is fine, we do not need to move it absolute value to the calc_exergy method. Subsequently we can use something like self.inl[0].Ex_physical - self.outl[0].Ex_physical for the component calculations.

Reply:
Alright, I agree.